test/unit/org/apache/cassandra/db/compaction/LeveledCompactionStrategyTest.java

```diff
@@ -823,4 +823,21 @@ private static void assertLevelsEqual(Collection<SSTableReader> l1, Collection<S
         assertEquals(l1.size(), l2.size());
         assertEquals(new HashSet<>(l1), new HashSet<>(l2));
     }
+
+    @Test(expected = RuntimeException.class)
```

Comment:
Only excepting a `RuntimeException` is not sufficient. 

`assertThatThrownBy` is a better alternative.
e.g. https://github.com/apache/cassandra/blob/27cc2fc3e275f56f1fa1df7285c389d5491acc8c/test/unit/org/apache/cassandra/db/marshal/EmptyTypeTest.java#L95-L100

Reply:
Good catch. Fixed.